walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +128,65 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsidered2() {
```

Comment:
naming, if you could please make it more descriptive.  If there was a reason to create a new test, this test I would hope covers a different case.  In which case, we can use that case to build the name here.